### DISCUSSION TITLE
Are pipelines now supported?

### DISCUSSION THREAD

**Discussion by nxorable** [oegedijk/explainerdashboard#164] [Q&A]
Per https://github.com/oegedijk/explainerdashboard/issues/11

...it appears we are waiting for a pipeline.get_feature_names() implementation to support `Pipeline`. Is this still the case, when we have something like a pipeline whose transformer expands the dimensionality prior to fitting with a classifier?

**Comment by oegedijk** [comment]
It seems that this feature is now available in sklearn v1, so should be able to add this yes!

https://scikit-learn.org/stable/modules/generated/sklearn.pipeline.Pipeline.html#sklearn.pipeline.Pipeline.get_feature_names_out

**Reply by nxorable** [reply]
That's good news! Any estimate on the date of this support enhancement?

**Reply by nxorable** [reply]
Friendly bump (as opposed to the unfriendly variety)

**Comment by oegedijk** [comment]
I've had zero time for this in the past two months (moving house with two young children is not exactly OSS development friendly :), but should have some time in the coming weeks so will see if I can do this.

If you'd like to help: basically just need a piece of code that take a Pipeline and extracts the dataframe that goes into the final model (with proper column names) and the final model.

**Reply by angelmtenor** [reply]
Here you can find a piece of code I use for that:

https://github.com/oegedijk/explainerdashboard/issues/213#issuecomment-1513362199
